english/00_8_Agent/The_name_is_Bond_Milton_Bond.md

```diff
@@ -0,0 +1,7 @@
+You have entered into the world of one of Her Majesty's secret agent. Codename Agent 008; his name is Bond, Milton Bond and he's the pencil pushing assistant of his more well-known cousin, [James](http://www.007.com/characters/the-bonds/). While James is out stopping evil-doers, rescusing damsels in distress, and having the occasional martini, Milton is responsible for making sure all of James' paperwork is in order.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.